app/_locales/en/messages.json

```diff
@@ -6327,7 +6327,7 @@
     "message": "Autodetect NFTs"
   },
   "useNftDetectionDescriptionText": {
-    "message": "Let MetaMask add NFTs you own using third-party services (like OpenSea). Autodetecting NFTs exposes your IP and account address to these services. Enabling this feature could associate your IP address with your Ethereum address and display fake NFTs airdropped by scammers. You can add tokens manually to avoid this risk."
```

Comment:
Hey @sahar-fehri actually you need to delete the message with this key in all other languages. Since the string is changed, we need to trigger a new translation job to the translation platform (Crowdin) and have all new translated strings pushed to another PR to ensure it is consistent in each language.

Reply:
Oh right, will follow up right away with another PR